channels/sync_runner_test.go

```diff
@@ -1,77 +1,76 @@
 	assertNotRejected(t, result)
 	result, _ = runner.Call(parse(`{}`), parse(`{"_roles": ["beta", "gamma"]}`), parse(`{"name": "", "roles": {"beta": ""}}`))
 	assertNotRejected(t, result)
 	result, _ = runner.Call(parse(`{}`), parse(`{"_roles": ["delta"]}`), parse(`{"name": "", "roles": {"beta":""}}`))
-	assertRejected(t, result, base.HTTPErrorf(403, base.SyncFnErrorMissingRole))
+	assertRejected(t, result, base.HTTPErrorf(http.StatusForbidden, base.SyncFnErrorMissingRole))
 }
 
 func TestRequireAccess(t *testing.T) {
 	const funcSource = `function(doc, oldDoc) { requireAccess(oldDoc._access) }`
 	runner, err := NewSyncRunner(funcSource)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	var result interface{}
 	result, _ = runner.Call(parse(`{}`), parse(`{"_access": ["alpha"]}`), parse(`{"name": "", "channels": ["alpha"]}`))
 	assertNotRejected(t, result)
 	result, _ = runner.Call(parse(`{}`), parse(`{"_access": ["beta", "gamma"]}`), parse(`{"name": "", "channels": ["beta"]}`))
 	assertNotRejected(t, result)
 	result, _ = runner.Call(parse(`{}`), parse(`{"_access": ["delta"]}`), parse(`{"name": "", "channels": ["beta"]}`))
-	assertRejected(t, result, base.HTTPErrorf(403, base.SyncFnErrorMissingChannelAccess))
+	assertRejected(t, result, base.HTTPErrorf(http.StatusForbidden, base.SyncFnErrorMissingChannelAccess))
 }
 
 func TestRequireAdmin(t *testing.T) {
 	const funcSource = `function(doc, oldDoc) { requireAdmin() }`
 	runner, err := NewSyncRunner(funcSource)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	var result interface{}
 	result, _ = runner.Call(parse(`{}`), parse(`{}`), parse(`{}`))
 	assertNotRejected(t, result)
 	result, _ = runner.Call(parse(`{}`), parse(`{}`), parse(`{"name": ""}`))
-	assertRejected(t, result, base.HTTPErrorf(403, base.SyncFnErrorAdminRequired))
+	assertRejected(t, result, base.HTTPErrorf(http.StatusForbidden, base.SyncFnErrorAdminRequired))
 	result, _ = runner.Call(parse(`{}`), parse(`{}`), parse(`{"name": "GUEST"}`))
-	assertRejected(t, result, base.HTTPErrorf(403, base.SyncFnErrorAdminRequired))
+	assertRejected(t, result, base.HTTPErrorf(http.StatusForbidden, base.SyncFnErrorAdminRequired))
 	result, _ = runner.Call(parse(`{}`), parse(`{}`), parse(`{"name": "beta"}`))
-	assertRejected(t, result, base.HTTPErrorf(403, base.SyncFnErrorAdminRequired))
+	assertRejected(t, result, base.HTTPErrorf(http.StatusForbidden, base.SyncFnErrorAdminRequired))
 }
 
 // Helpers
 func assertRejected(t *testing.T, result interface{}, err *base.HTTPError) {
 	r, ok := result.(*ChannelMapperOutput)
-	goassert.True(t, ok)
-	goassert.DeepEquals(t, r.Rejection, err)
+	require.True(t, ok)
+	require.Equal(t, r.Rejection, err)
 }
 
 func assertNotRejected(t *testing.T, result interface{}) {
 	r, ok := result.(*ChannelMapperOutput)
-	if !ok || r.Rejection != nil {
-		t.Fatalf("%v", r.Rejection)
-	}
+	require.True(t, ok)
+	require.NoError(t, r.Rejection)
```

Comment:
It looks like these should still be asserts, at least based on the usage in the tests above.

Reply:
Yeah, that makes sense.